src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -261,12 +269,131 @@ private static File createInOrderSampleMap() {
         return getSampleMapFile(sampleFileContents);
     }
 
-    private static File getSampleMapFile(String sampleFileContents) {
+    private static File getSampleMapFile(final String sampleFileContents) {
         final File sampleNameMap = IOUtils.writeTempFile(sampleFileContents, "sampleNameMap", ".txt");
         sampleNameMap.deleteOnExit();
         return sampleNameMap;
     }
 
+    private static File getSampleMapFile(final Map<String, String> mapping){
+        return getSampleMapFile(mapping.entrySet()
+                .stream()
+                .map( pair -> pair.getKey() + "\t" + pair.getValue())
+                .collect(Collectors.joining("\n")));
+    }
+
+    @DataProvider
+    public static Iterator<Object[]> getRenameCombinations() {
+        final Map<String,String> noRemapping = new LinkedHashMap<>();
+        noRemapping.put("s1", "s1");
+        noRemapping.put("s2", "s2");
+        noRemapping.put("s3", "s3");
```

Comment:
We really could use a test case with more than a handful of samples -- say, 100. You could write a method to permute the values in a sample -> sample mapping.

Reply:
opened #3861